packages/jellyfish-api-core/__tests__/category/governance/governance_container.ts

```diff
@@ -2,23 +2,18 @@ import { MasterNodeRegTestContainer, StartOptions } from '@defichain/testcontain
 
 export class GovernanceMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
   constructor () {
-    super(undefined, 'defi/defichain:master-bcce2b47e')
+    super(undefined, 'defi/defichain:epic-grandcentral-493dc64b6')
```

Comment:
wasn't sure if I should update the image here or in `DeFiDContainer`, updated it here for now